README.md

```diff
@@ -62,7 +62,7 @@ Returns a new focus trap on `element`.
 - **clickOutsideDeactivates** {boolean}: Default: `false`. If `true`, a click outside the focus trap will deactivate the focus trap and allow the click event to do its thing.
 - **returnFocusOnDeactivate** {boolean}: Default: `true`. If `false`, when the trap is deactivated, focus will *not* return to the element that had focus before activation.
 
-### focusTrap.activate()
+### focusTrap.activate([activateOptions])
```

Comment:
Would you mind adding a description of the `onActivate` option, mimicking the formatting of the `deactivateOptions`? Looks like that's missing, too.

Reply:
Sure thing! Done ✓.